packages/react-ui-map/src/Mapbox/Mapbox.js

```diff
@@ -83,17 +89,19 @@ export default class Mapbox extends PureComponent {
     const { map } = this.state
 
     return (
-      <div
-        style={{}}
-        id="map"
-        className={classNames(
-          className,
-          theme[`Map-${color}`],
-          theme[`Map-${size}`],
-          theme.Map
-        )}
-      >
-        {map && children}
+      <div>
+        <div
+          ref="map"
```

Comment:
This is the incorrect usage of a `ref`. Please refer to the [docs for usage](https://facebook.github.io/react/docs/refs-and-the-dom.html#adding-a-ref-to-a-dom-element)